wiki/Beatmap_Editor/Song_Setup/Primary_metadata_source/en.md

```diff
@@ -0,0 +1,16 @@
+---
+tags:
+  - metadata
+  - song metadata
+  - source
+---
+
+# Primary metadata source
+
+An **primary metadata source** is a source for `Artist`, `Title`, and `Source` fields in a [beatmapset's metadata](/wiki/Beatmap_Editor/Song_Setup#song-and-map-metadata) officially promoted by the artist or their label.
+
+An artist's website, their label's website, CD scans, or official uploads to websites like Bandcamp, YouTube, or SoundCloud are considered primary sources for song metadata. Third party websites such as wikis, databases (vndb, vgmdb, etc.), or music services (Spotify, iTunes, Amazon, etc.) are unreliable and therefore not used for song metadata.
```

Comment:
> Third party websites [...] are unreliable

sounds kinda harsh (maybe by accident), the common wikis and databases around just don't follow the same rules that osu! does with metadata.
```suggestion
An artist's website, their label's website, CD scans, and official uploads to websites like Bandcamp, YouTube, and SoundCloud are considered primary sources for song metadata. Third party websites such as wikis, databases (vndb, vgmdb, etc.), and music services (Spotify, iTunes, Amazon, etc.) do not always reflect the artist's intention and should not be treated as primary sources.
```